src/shared/components/app/navbar.tsx

```diff
@@ -220,42 +220,51 @@ export class Navbar extends Component<NavbarProps, NavbarState> {
             )}
             <li className="nav-item">
               <a
-                className="nav-link"
+                className="nav-link d-inline-flex align-items-center d-md-inline-block"
```

Comment:
Nitpick: since these classnames are pretty long and also repeated, it could be worth extracting into a variable.